docs/user/openstack/deploy_on_ovs_dpdk.md

```diff
@@ -0,0 +1,94 @@
+# Deploying an OpenShift Cluster onto OpenStack with OVS-DPDK
+
+
+An OpenStack deployment with ovs-dpdk enabled provides network performance improvements.
+
+Deploying an OpenShift cluster onto an Openstack deployment with ovs-dpdk, is a straight forward process. 
+
+## Selecting hugetable size and cpu policy for cluster vms.
+When deploying vms on to this type of PaaS, the correct hugetable size and cpu pinnings need to be configured for the vms. 
+
+
+This maybe accomplished in two ways: 
+
+
+2) The flavors used for the OCP are defined with appropriate values. For example: 
+   ```
+   $opensack openstack flavor create dpdk_flavor --ram 16384 --disk 45 --vcpus 4
+   $ openstack flavor set \
+    --property dpdk=true \
+    --property hw:cpu_policy=dedicated \
+    --property hw:mem_page_size=large \
+    dpdk_flavor.small
+   ```
+
+3) The image properties maybe be defined in the install-config.yaml using [platform.openstack.
+   clusterOSImageProperties][customization.md]. For example:
+   ```
+   platform:
+      openstack:
+         clusterOSImageProperties:
+            - hw_cpu_policy: dedicated
+            - hw_mem_page_size: 1GB
```

Comment:
You used `large` above but `1GB` above. I suggest using the same in both places.

Later: oh, actually, maybe you need to specify an exact size for image metadata properties?

Reply:
actually in one place it needs to be 1GB in in the other it's either Large, small or any. At least that's what the openstack docs say. We should test. 
The image property takes a number but the flavor property takes a word.